.gitmodules

```diff
@@ -10,3 +10,9 @@
 	path = contracts/lib/bn254
 	url = https://github.com/EspressoSystems/solidity-bn254
 	branch = v0.1.1
+[submodule "contracts/lib/openzeppelin-contracts"]
```

Comment:
@philippecamacho I think this can be removed now. For the future it seems that foundry cannot handle multiple contracts with the same name being used in the project. The problem is that when creating bindings the bindings aren't namespaced so there's only one ERC20 token in the bindings. The bindings work anyway because the contracts should have the same interface. The problems are that that the 

1. The bytecode is embedded in the bindings and one seems to be chosen at random, it might not be exactly the same between different implementations.
2. The bindings files for the contracts also have some other differences. This is why we were sometimes seeing the CI job that checks the bindings fail because of a different ERC20 bindings file.

If we really need to have implementations of a contract with the same name we can probably do some workarounds to make it work but it would require some work. I think we don't need it at the moment.

cc @alxiong

Reply:
Done in 44fc0f0.